extensions/oidc/runtime/src/main/java/io/quarkus/oidc/runtime/OidcProviderClientImpl.java

```diff
@@ -338,9 +338,11 @@ private UniOnItem<HttpResponse<Buffer>> getHttpResponse(OidcRequestContextProper
                 request.putHeader(headerEntry.getKey(), headerEntry.getValue());
             }
         }
-
-        LOG.debugf("%s token: %s params: %s headers: %s", (introspect ? "Introspect" : "Get"), metadata.getTokenUri(), formBody,
-                request.headers());
+        if (LOG.isDebugEnabled()) {
+            LOG.debugf("%s token: %s params: %s headers: %s", (introspect ? "Introspect" : "Get"), metadata.getTokenUri(),
```

Comment:
> (introspect ? "Introspect" : "Get")

I think this method can also be used to refresh/revoke token? Using "Get" for the refresh is fine with me, but the using "Get token" for revoking token is bit confusing?

Reply:
@michalvavrik Michal, would you mind id I do a separate PR immediately after this one is merged to clean up the `quarkus-oidc` code ? I found some bug there (Token revocation only filters don't work in `quarkus-oidc` do not work, not a big prob as a filter catching all requests can still get them), I'd like to add a test, so it will all become a bit bigger  than updating OIDC Client only to log requests